README.md

```diff
@@ -200,6 +218,10 @@ function onloadCallback() {
 It is crucial that `api.js` is loaded before the widget is instantiated. Therefore we are using a callback function to load the widget. The `grecaptcha` object
 will be available after the `api.js` is loaded. For more information, see [Captcha documentations](https://developers.google.com/recaptcha/docs/display).
 
+## Using Risk-Based Authentication With the Widget
```

Comment:
The ToC link can navigate to the new file.

Reply:
Changed.